Makefile

```diff
@@ -79,8 +80,8 @@ install:
 # tests.
 .PHONY: test
 test:
-	$(GO) test -tags '$(TAGS)' $(GOFLAGS) -i $(PKG)
-	$(GO) test -tags '$(TAGS)' $(GOFLAGS) -run $(TESTS) $(PKG) -timeout $(TESTTIMEOUT) $(TESTFLAGS)
+	$(GO) test -cpu $(CPUS) -tags '$(TAGS)' $(GOFLAGS) -i $(PKG)
```

Comment:
I don't think `-cpu` does anything with `-i` (which builds and installs the tests without running them)